pkg/localize/locales/en/cmd/cluster.en.toml

```diff
@@ -301,11 +305,21 @@ Client ID:     {{.ClientID}}
 Make a copy of the client ID to store in a safe place. Credentials won't appear again after closing the terminal.
 
 You will need to assign permissions to service account in order to use it. 
-For example for Kafka service you should execute the following command to grant access to the service account:
+'''
+
+[cluster.kubernetes.printKafkaAccessCommands]
+one = '''
+For Kafka service you should execute the following command to grant access to the service account:
```

Comment:
After verifying I found this quite odd message.
I'm not sure if there is benefit of mentioning service name here due

I will change it to something like 

```
You need to grant access to you Kafka separately by issuing following command
```